test_scripts/UnknownEnums/002_unknown_nonmandatory_param.lua

```diff
@@ -0,0 +1,70 @@
+---------------------------------------------------------------------------------------------------
+-- Description:
+-- Mobile sends an unknown enum as a non-mandatory param. The enum will be filtered out and
+-- WARNINGS will be returned because the parameter was not mandatory
+
+-- Pre-conditions:
+-- a. HMI and SDL are started
+-- b. appID is registered and activated on SDL
+
+-- Steps:
+-- appID requests SetMediaClockTimer with an unknown audioStreamingIndicator enum
+
+-- Expected:
+-- SDL Core filters out the unknown enum. The HMI receives the request without the unknown enum.
+-- WARNINGS result is returned to mobile.
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local commonSmoke = require('test_scripts/Smoke/commonSmoke')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local requestParams = {
+    updateMode = "COUNTUP",
+    startTime = {
+        hours = 0,
+        minutes = 0,
+        seconds = 0
+    },
+    audioStreamingIndicator = "UNKNOWN"
+}
+
+local hmiRequestParams = {
```

Comment:
Sadly I don't believe that this verifies that the value did not reach the HMI. You might need to add a custom `ValidIf` function to truly verify that. This would affect several of these tests